doc/source/examples/multiphysics/bubble-detachment-shear-flow/bubble-detachment-shear-flow.rst

```diff
@@ -0,0 +1,437 @@
+
+Then, to post-process the results, you may use the provided ``multiple_folders_bubble_detachment_post_processing.py`` python script. It contains all the instructions inside to run it.
+
+
+-----------------
+Results
+-----------------
+
+In order to analyze the influence of the surrounding liquid on the detachment of the bubble, we run the simulations for different values of the shear rate: :math:`S \in [100,200,300,450]`. The detachment time and volume are then computed and compared to the results of Mirsandi *et al.* [#mirsandi2020]_ in the following figure, which shows an excellent agreement. The result of the no-shear simulation was added to the plot for completion.
+
++-------------------------------------------------------------------------------------------------------------------+
+|  .. figure:: images/bubble-detachment_volume.png                                                                  |
+|     :alt: Plot of the detachment volume (our results and literature results) with respect to shear rate.          |
+|     :align: center                                                                                                |
+|     :name: Detachment volumes                                                                                     |
+|                                                                                                                   |
+|     Plot of the detachment volume (our results and literature results) with respect to shear rate.                |
+|                                                                                                                   |
++-------------------------------------------------------------------------------------------------------------------+
+
++-------------------------------------------------------------------------------------------------------------------+
+|  .. figure:: images/bubble-detachment_time.png                                                                    |
+|     :alt: Plot of the detachment time (our results and literature results) with respect to shear rate.            |
+|     :align: center                                                                                                |
+|     :name: Detachment times                                                                                       |
+|                                                                                                                   |
+|     Plot of the detachment times (our results and literature results) with respect to shear rate.                 |
+|                                                                                                                   |
++-------------------------------------------------------------------------------------------------------------------+
+
+Below are the plots of the contour of the bubble in the plane :math:`z = 0` when detachment occurs for different values of the shear rate.
```

Comment:
Do you want to add one of your video?

Reply:
Yes! I still need to make them prettier with @AmishgaAlphonius though